bitcoin/src/blockdata/transaction.rs

```diff
@@ -265,7 +265,7 @@ pub trait TransactionExt: sealed::Sealed {
     /// > Virtual transaction size is defined as Transaction weight / 4 (rounded up to the next integer).
     ///
     /// [`BIP141`]: https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki
-    /// [`policy`]: ../../policy/index.html
+    /// [`policy`]: ../policy/index.html
```

Comment:
```suggestion
    /// [`policy`]: super::policy
```

Reply:
I just tried that with rust 1.85. Cargo doc gives the following error, and the link doesn't work:

```
 Documenting bitcoin v0.33.0-alpha.0 (/home/user/rust-bitcoin/bitcoin)
warning: unresolved link to `super::policy`
   --> bitcoin/src/blockdata/transaction.rs:268:21
    |
268 |     /// [`policy`]: super::policy
    |                     ^^^^^^^^^^^^^ no item named `policy` in module `blockdata`
    |
    = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default

warning: `bitcoin` (lib doc) generated 1 warning
```

Reply:
Oh, the correct path seems to be `crate::policy`, not `super::policy`.

Reply:
That worked; pushed a new version.